pkg/loadbalancer/server_test.go

```diff
@@ -20,8 +20,8 @@ func TestLoadBalancerName(t *testing.T) {
 			name:        "simple",
 			cluster:     "test-cluster",
 			service:     &v1.Service{ObjectMeta: metav1.ObjectMeta{Namespace: "test-namespace", Name: "test-service"}},
-			expected:    constants.ContainerPrefix + "-CGVXJAVBASN2Z3RXOABMYVHNP7WNHR3ATSDVOTEN",
```

Comment:
a cloud-provider-kind delete the old loadbalancers on exit and create new ones, so this change should not break existing users